CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,96 @@
+# Gitea Community Code of Conduct
+
+## About
+
+Online communities include people from many different backgrounds. The Gitea contributors are committed to providing a friendly, safe and welcoming environment for all, regardless of gender identity and expression, sexual orientation, disabilities, neurodiversity, physical appearance, body size, ethnicity, nationality, race, age, religion, or similar personal characteristics.
+
+The first goal of the Code of Conduct is to specify a baseline standard of behavior so that people with different social values and communication styles can talk about Gitea effectively, productively, and respectfully.
+
+The second goal is to provide a mechanism for resolving conflicts in the community when they arise.
+
+The third goal of the Code of Conduct is to make our community welcoming to people from different backgrounds. Diversity is critical to the project; for Gitea to be successful, it needs contributors and users from all backgrounds.
+
+We believe that healthy debate and disagreement are essential to a healthy project and community. However, it is never ok to be disrespectful. We value diverse opinions, but we value respectful behavior more.
+
+## Giteaer values
```

Comment:
`Giteaer`?
Sounds weird, especially since there are three subsequent vocals, which always looks strange.

Reply:
I think so. Any suggestion?

Reply:
`Community values` or similar imo. In the original this was `Gopher` as that is the nickname for Go devs, however I don't think we have such a nickname, so here it is largely irrelevant.

Reply:
:+1: for `Community values`.
```suggestion
## Community values
```